core/snippets/snippets/javascript.snippet

```diff
@@ -0,0 +1,56 @@
+language: javascript
+---
+
+name: forInLoopStatement
+phrase: for in
+insertionScope: statement
+-
+for (const $1 in $2) {
+    $0
+}
+---
+
+name: anonymousFunctionDeclarationAndCall
+phrase: self calling
+-
+(() => {
+    $0
+})();
+---
+
+name: importStarStatement
+phrase: import star
+-
+import * as $0 from "$0";
+---
+
+name: importFromStatement
```

Comment:
we can tweak in follow-up, but prob useful to define this one in its own file so we can define for Python as well

Reply:
True